iocore/net/SSLClientUtils.cc

```diff
@@ -138,6 +138,20 @@ verify_callback(int signature_ok, X509_STORE_CTX *ctx)
   return true;
 }
 
+static int
+ssl_client_cert_callback(SSL *ssl, void * /*arg*/)
+{
+  SSLNetVConnection *netvc = SSLNetVCAccess(ssl);
```

Comment:
A new callback function just for SSLNetVC 😢  Please create an issue for QUIC.

Reply:
Sorry I don't see where to put this in the QUIC classes. Filed issue #6939.